dhis-2/dhis-services/dhis-service-core/pom.xml

```diff
@@ -185,9 +185,9 @@
       <artifactId>commons-collections</artifactId>
     </dependency>
     <dependency>
-      <groupId>com.github.dhis2</groupId>
+      <groupId>com.github.JasperTimm</groupId>
```

Comment:
@JasperTimm I have mered the PR for the sms-compression library. So I think we don't need to change this reference.

Reply:
@zubaira  - yep, updated to the dhis2 repo release now, thanks. Please keep in mind this PR includes the fix in #5159, so we should merge that first.